python/tank/platform/engine.py

```diff
@@ -441,6 +440,8 @@ def destroy(self):
         # explicitly set the value to None!
         if self._invoker:
             self._invoker = None
+        if self._async_invoker:
```

Comment:
Minor: Wondering why we do the none check here - maybe it should just be set to none straight away?